datafusion/expr/src/test/function_stub.rs

```diff
@@ -43,15 +43,13 @@ macro_rules! create_func {
             /// Singleton instance of [$UDAF], ensures the UDAF is only created once
             /// named STATIC_$(UDAF). For example `STATIC_FirstValue`
             #[allow(non_upper_case_globals)]
-            static [< STATIC_ $UDAF >]: std::sync::OnceLock<std::sync::Arc<crate::AggregateUDF>> =
```

Comment:
The reason why we have full path in macro is that we can avoid repeatedly importing where ever we have macro

Reply:
I know, but this macro isn't public.

Reply:
removed anyway.